providers/standard/tests/provider_tests/standard/operators/test_generic_transfer.py

```diff
@@ -26,6 +26,7 @@
 
 from airflow.exceptions import AirflowProviderDeprecationWarning
 from airflow.models.dag import DAG
+from airflow.providers.mysql.hooks.mysql import MySqlHook
```

Comment:
Still creates a dependency between providers right?
I mean should be OK but ideally, we can move to common

Reply:
Fix with move to `tests_common` here: https://github.com/apache/airflow/pull/46256

Reply:
Just saw your comment, feel free to proceed with this one or if you think my PR does a better job, lets proceed with that!

Reply:
No. 

* It's ok to have dependency to other providers. src/ of each provider is on the path and you can import `from airflowproviders.OTHER_PROVIDER` - this is the same as if you have other providers installed from a package.

* It's NOT ok to have `from providers.test.*` - > this is test code for the other provider, it's not supposed to be imported from outside of this provider tests.